File: Modul 5/soal_5/PRAK505-2410817310003-Muhammad_Guntur_Ricky_Adhitya.py
def Biodata(tahunLahir, nama, asal):
    tahun_sekarang = 2020
    print("Perkenalkan Nama Saya : %s"%(nama))
    print("Umur Saya : %d"%(tahun_sekarang-tahunLahir))
    print("Saya Adalah Angkatan : 2020")
    print("Asal Saya dari : %s"%(asal))

tahunLahir = int(input())
nama = input()
asal = input()
Biodata(tahunLahir, nama, asal)

File: Modul 5/soal_5/PRAK505-2410817310003-Muhammad_Guntur_Ricky_Adhitya.c
#include <stdio.h>

void Biodata(int tahunLahir, char nama[], char asal[]){
    int tahun_sekarang = 2020;
    printf("Perkenalkan Nama Saya : %s\n", nama);
    printf("Umur Saya : %d\n", tahun_sekarang-tahunLahir);
    printf("Saya Adalah Angkatan : 2020\n");
    printf("Asal Saya dari : %s\n", asal);
}

int main() {
    int tahunLahir;
    char A[20], B[15];
    scanf(" %d",&tahunLahir);
    scanf(" %[^\n]%*c",&A);
    scanf(" %[^\n]%*c",&B);
    Biodata(tahunLahir, A, B);
    return 0;
}